app/models/reports/matview.rb

```diff
@@ -4,7 +4,29 @@ def self.refresh
       ActiveRecord::Base.transaction do
         ActiveRecord::Base.connection.execute("SET LOCAL TIME ZONE '#{Period::REPORTING_TIME_ZONE}'")
         Scenic.database.refresh_materialized_view(table_name, concurrently: true, cascade: false)
+        add_comments
       end
     end
+
+    def self.add_comments
+      table_comments = YAML.load_file("config/schema_descriptions.yml")[table_name]
+      return unless table_comments.present?
+      add_table_description(table_name, table_comments["description"])
+      table_comments["columns"].each do |column_name, column_description|
+        add_column_description(column_description, column_name)
+      end
+    end
+
+    def self.add_table_description(table_name, table_description)
```

Comment:
Rails doesn't have a way to change comments from what I can tell, some [older gems](https://github.com/pinnymz/migration_comments) do. I'll try them out.

I would also like the ability to throw an error if the yml file doesn't have the description for a given column, and we'd need to hand-roll that at least.

Reply:
Re: throwing errors. I like that idea. It reminds me of the Sentry alerts we had for "regions integrity check", if any parts of our region hierarchy are out of whack. This would be a similar reminder for us to populate missing descriptions if we forget during PR review.

Reply:
If we want to autogenerate this "handbook", shall we modify the YML structure to tag each field as, say, "important" and "not important", or something? Some mechanism that'll allow us to highlight the main fields that we think will be used often, and put the other verbose stuff somewhere below?

Reply:
Oh yes, the "summary" vs "detail" distinction seems useful. I'll add that in!